de.tumexam.cli.yml

```diff
@@ -135,7 +134,7 @@ modules:
                 sources:
                   - type: git
                     url: https://gitlab.gnome.org/GNOME/glibmm.git
-                    tag: 2.70.0
-                    commit: 042e90eebca380686bbcc24d6803305b9d2c7823
+                    tag: 2.74.0
```

Comment:
you should stick to 2.72.x

Reply:
But I depend on 2.74.0 features.

Reply:
nevermind. 

earlier build failure was because 2.72 was detected. Some stream may have crossed since this did build (and yes 2.74.x is in the SDK).

Reply:
ACK, thanks!